components/src/core/components/List/List.vue

```diff
@@ -268,7 +268,9 @@ export default defineComponent({
 
     const filterTitle = computed((): string => {
       const itemCount =
-        state.selectedItemIndexes.length || props.filteredTotalRecordsCount;
+        props.filteredTotalRecordsCount > 0 ?
+          props.filteredTotalRecordsCount :
```

Comment:
Is this change really needed? logical OR operator should return total count if length is 0.
```
0 || 42 => 42
undefined || 42 => 42
```